microsoft-edge/toc.yml

```diff
@@ -1414,17 +1414,17 @@
                   href: webview2/how-to/remote-debugging.md
```

Comment:
To consider: I think we could remove this page. All the content is already in the other pages, and TOC is already on the left side navigation.

Reply:
keeping nav page, b/c several articles link to it as a platform-neutral link destination.